Update latest event video

inferiorolive opened the pull request:
This update added new video metadata for latest event video display

[Link to issue](https://github.com/chicagoruby/chicagoruby-website/issues/9)

**Change**

<img width="700" height: auto alt="Screenshot 2025-06-06 at 18 15 53" src="https://github.com/user-attachments/assets/8cfe9a61-4f08-439d-8a7b-b606bed4b95b" />
<img width="700" height: auto alt="Screenshot 2025-06-06 at 18 15 53" src="https://github.com/user-attachments/assets/410617d5-224d-4f71-8c85-e3ad3ee58ed7" />

inferiorolive committed:
Add May 2025 ChicagoRuby meetup video metadata

inferiorolive committed:
Fix youtube ID

yuenmichelle1 reviewed (approved):
LGTM! 

This can be in a different PR or bundled with this PR, mind adding the other videos from ChicagoRuby events from march and April as well? 


videos can be found here: https://youtube.com/@chicagorubymeetups?si=Kv6MEh4gc4M8bRFV

inferiorolive committed:
Remove repeated authors from latest video title

inferiorolive committed:
Add April 2025 ChicagoRuby meetup video metadata

inferiorolive committed:
Add May 2025 ChicagoRuby meetup video metadata

inferiorolive committed:
Remove accidental comment

inferiorolive committed:
Update title of Modular Monoliths post

inferiorolive commented:
Added the two most recent videos from Chicago Ruby's meetup channel 
Fixed repeating author names in latest video title  

**March**
![Screenshot 2025-06-19 at 22 45 30](https://github.com/user-attachments/assets/f860e7ea-0e80-4e20-a66a-82df22f22e7e)

**April**
![Screenshot 2025-06-19 at 22 48 39](https://github.com/user-attachments/assets/70990a27-659f-48c5-971e-40c1d6a15bbe)

**May**
![Screenshot 2025-06-19 at 22 48 10](https://github.com/user-attachments/assets/92867361-32ba-4666-99bb-462d188db97a)

inferiorolive commented:
@yuenmichelle1 ready for ya :)

yuenmichelle1 commented:
@inferiorolive YAAaaasss!! Amazing! Imma go ahead and merge 🤩🤩

Good stuff! 🌟
